crates/apollo-parser/CHANGELOG.md

```diff
@@ -17,6 +17,19 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Documentation -->
 
+# [0.8.0](https://crates.io/crates/apollo-parser/0.8.0) - 2024-07-30
+
+This release removes the `Error::new` constructor. We recommend not creating instances of
+`apollo_parser::Error` yourself at all, but this is a breaking change.
```

Comment:
I guess so. I didn't want to pull too much attention to it because I think it will affect 0 users. Let's do it for consistency with other release notes.
```suggestion
## BREAKING
This release removes the `Error::new` constructor. We recommend not creating instances of
`apollo_parser::Error` yourself at all.
```